### PR TITLE
CI/CD build

### PR DESCRIPTION
**Pull request by adidoshiws**
### Ticket Link

---

### Related Links

---

### Description
- CD pipeline 
- Updated the cd.yml workflow
Checkout -> Install -> Build -> Sync -> Cache
- Uploaded secret keys - environment variables

### Steps to Reproduce / Test

---

### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### Live Link

http://aditya-react-challenge.s3-website.ap-south-1.amazonaws.com/cd-pipeline/tracks

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -5,42 +5,38 @@ on:
...
-          token: ${{ secrets.GITHUB_TOKEN }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.AWS_REGION }}
 
-      - name: Install dependencies
+      - name: Install Node.js 12.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
```
Comment (tousifws) [inline .github/workflows/cd.yml]: The minimum node version should be 14.x https://endoflife.date/nodejs

File: .github/workflows/uat.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
       matrix:
         node-version: [16.x]
     env:
-      AWS_S3_BUCKET: wednesday-react-template
+      AWS_S3_BUCKET: aditya-react-challenge
```
Comment (tousifws) [inline .github/workflows/uat.yml]: Get it from GitHub secrets also

**Review by tousifws — COMMENTED** [review]
Please put the live link in the comment. And set the annotation of jest coverage action to failed-tests. It makes it super annoying to review. https://github.com/ArtiomTr/jest-coverage-report-action#change-annotations

File: .github/workflows/uat.yml (hunk 2 above)
Comment (adidoshiws) [reply .github/workflows/uat.yml]: Done

File: .github/workflows/cd.yml (hunk 1 above)
Comment (adidoshiws) [reply .github/workflows/cd.yml]: Updated

**Review by ritwik-wednesday — APPROVED** [review]

**Pull request merged**